dll/cpl/desk/desktop.c

```diff
@@ -24,11 +36,30 @@ struct
     {TEXT("{871C5380-42A0-1069-A2EA-08002B30309D}"), IDC_ICONS_INTERNET}, /* Internet Browser */
 };
 
+static const TCHAR szUserClass[] = TEXT("Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\CLSID\\");
+static const TCHAR szSysClass[] = TEXT("CLSID\\");
+static const TCHAR szDefaultIcon[] = TEXT("\\DefaultIcon");
+static const TCHAR szFallbackIcon[] = TEXT("%SystemRoot%\\system32\\shell32.dll,0");
+
+struct
+{
+    LPCTSTR CLSID;
+    UINT Title;
+    LPCTSTR Icon;
+} IconChange[NUM_CHANGE_ICONS] = {
+    {TEXT("{20D04FE0-3AEA-1069-A2D8-08002B30309D}"), IDS_TITLE_MYCOMP, NULL},         /* My Computer */
+    {TEXT("{450D8FBA-AD25-11D0-98A8-0800361B1103}"), IDS_PERSONAL, NULL},             /* My Documents */
+    {TEXT("{208D2C60-3AEA-1069-A2D7-08002B30309D}"), IDS_TITLE_MYNET, NULL},          /* My Network Places */
+    {TEXT("{645FF040-5081-101B-9F08-00AA002F954E}"), IDS_TITLE_BIN_1, TEXT("Full")},  /* Recycle Bin (full) */
+    {TEXT("{645FF040-5081-101B-9F08-00AA002F954E}"), IDS_TITLE_BIN_0, TEXT("Empty")}, /* Recycle Bin (empty) */
+};
```

Comment:
TBH It doesn't make sense to write new code with TCHAR

Reply:
I expected these comments as I saw them in other PR :smile: I want to keep consistency with existing code in this file.